LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/UdpServer.cs

```diff
@@ -204,10 +204,19 @@ private async Task ProcessRxpkAsync(String remoteIp, Rxpk rxpk, DateTime startTi
                     var messageByte = Encoding.UTF8.GetBytes(jsonMsg);
                     var token = await GetTokenAsync();
                     PhysicalPayload pyld = new PhysicalPayload(token, PhysicalIdentifier.PULL_RESP, messageByte);
-                    await this.UdpSendMessage(pyld.GetMessage(), remoteIp, pullAckRemoteLoRaAggregatorPort);
-                    Logger.Log("UDP", String.Format("message sent with ID {0}",
-                        ConversionHelper.ByteArrayToString(token)),
-                        Logger.LoggingLevel.Info);
+                    if (pullAckRemoteLoRaAggregatorPort != 0)
+                    {
+                        await this.UdpSendMessage(pyld.GetMessage(), remoteIp, pullAckRemoteLoRaAggregatorPort);
+                        Logger.Log("UDP", String.Format("message sent with ID {0}",
+                            ConversionHelper.ByteArrayToString(token)),
+                            Logger.LoggingLevel.Info);
+                    }
+                    else
+                    {
+                        Logger.Log("UDP", 
```

Comment:
Are we sure that we want to log as error? I think this can happen so should not be an error

Reply:
I felt this could lead to some confusion, so decided to always log it. Happy to revert to Info if you think it's more appropriate.

Reply:
Info is better, if not it would alarm the user for a startup inconvenient that we can do nothing about